tests/test_wps_climdex_mmdmt.py

```diff
@@ -23,13 +23,13 @@ def build_params(climdex_input, ci_name, month_type, freq, vector_name, output_f
         (local_path("climdexInput.rda"), "ci", "txx", "annual", "vector_name"),
         (local_path("climdexInput.rda"), "ci", "tnx", "monthly", "vector_name"),
         (local_path("climdexInput.rda"), "ci", "tnx", "annual", "vector_name"),
-        (local_path("climdexInput.rda"), "ci", "txn", "monthly", "vector_name"),
-        (local_path("climdexInput.rda"), "ci", "txn", "annual", "vector_name"),
-        (local_path("climdexInput.rda"), "ci", "tnn", "monthly", "vector_name"),
-        (local_path("climdexInput.rda"), "ci", "tnn", "annual", "vector_name"),
+        (local_path("climdexInput.rds"), "ci", "txn", "monthly", "vector_name"),
+        (local_path("climdexInput.rds"), "ci", "txn", "annual", "vector_name"),
+        (local_path("climdexInput.rds"), "ci", "tnn", "monthly", "vector_name"),
+        (local_path("climdexInput.rds"), "ci", "tnn", "annual", "vector_name"),
     ],
 )
-def test_wps_climdex_mmdmt(climdex_input, ci_name, month_type, freq, vector_name):
+def test_wps_climdex_mmdmt_rda(climdex_input, ci_name, month_type, freq, vector_name):
```

Comment:
Why change the name here? We seem to be using both `rds` and `rda` files.

Reply:
That was left over from when I had the two inputs. I will change it back.